### PR TITLE
Got Add Incident Modal to Appear When Clicking Create Incident 

### PR DESCRIPTION
**Pull request by ahmedseragcodes**
#Loom Video Review Link

UI Changes: https://www.loom.com/share/1a81dd7f53b14a2f85634b12cf80074f
Code Changes: https://www.loom.com/share/44d8e60e2b7646259225eea8cb915ba5

##Description

Previously when you clicked create incident on the approved incidents list view within the admin dashboard, the add incident modal wasn't appearing. I simply copied over that functionality from the unapproved incidents list view and now it works for the approved incidents list view / ie the modal appears when you click create incident.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Change Status

Complete, needs unit tests 

# How Has This Been Tested?

Repeatedly testing UI functionality, also a copy of previously working code, just re-using it somewhere new 

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by gaben3722 — COMMENTED** [review]
Everything looks good to me!

**Review by gaben3722 — APPROVED** [review]

**Comment by austin-carman** [comment]
Looks good to me as well. Good description and loom video shows the functionality

**Review by austin-carman — APPROVED** [review]
Looks good. Good description and loom video shows functionality

File: src/components/graphs/bargraph/HorizontalBar.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ const Horizontalbar = () => {
 
   return (
     <div>
+      {console.log('LOOK HERE', dataList)}
```
Comment (paulstgermain) [inline src/components/graphs/bargraph/HorizontalBar.js]: Is this `console.log` strictly necessary to keep in the code now that we have the feature implemented?

**Review by paulstgermain — CHANGES_REQUESTED** [review]
## Video Feedback

Your walkthrough of the feature you implemented, the changes it brought to the application, and the problem it solves were great here! Both succinct and detailed, I can see exactly why you wrote the code you did. That said, you didn't walk us through the _how_! Can you record a second video walking through your PR, the code you wrote, and how it works?

## Code Review

The code you added here is simple, and clean - clearly implementing a feature we already have in place in the 'Unapproved Incidents' tab into the 'Approved Incidents Tab'. This clearly improves user experience by ensuring that no matter which tab our Admins are looking at they can add an incident as intended originally.

That said, there is the question of the lingering `console.log` in **HorizontalBar.js** - is this something we still need? If not, it should be removed before we merge this into main.

Please record a new video walking through the code, and either resolve the open conversation regarding the `console.log` with an explanation as to why we need it, or remove it and push that new commit up before we merge this to main.

File: src/components/graphs/bargraph/HorizontalBar.js (hunk 1 above)
Comment (ahmedseragcodes) [reply src/components/graphs/bargraph/HorizontalBar.js]: Yep that's removed with the latest commit now

**Comment by ahmedseragcodes** [comment]
Removed left-behind console log + added the second loom explaining in-code changes

**Comment by paulstgermain** [comment]
## Code Walkthrough Review

Thank you for putting in some time to record yet another walkthrough! Your explanation of the changes made in this PR was succinct, yet detailed enough that it made the changes you made as well as the reasons for them very clear - good work! My only feedback on this at this time is to attempt to slow down your explanations for your listeners in the future; your interviewers may not have the same level of technical knowledge, and it may be more difficult for them to understand what you're explaining.

For these and the reasons listed above, I'm approving and merging this PR!

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**